client/src/views/page-components/account/history/history.scss

```
@@ -0,0 +1,61 @@
+.history__header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 25px;
+}
+
+.history__header-left {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+.history__text {
+  font-size: 30px;
+}
+
+.history__add {
+  font-size: 50px;
+}
+
+.table {
+  table-layout: fixed;
+  width: 100%;
+  border-collapse: separate;
+  border-spacing: 0 10px;
+  // 첫줄은 테이블 고정
+  // 아래 2줄은 테이블 간격
+}
+
+.table__content-container {
+  display: flex;
+  gap: 20px;
+  align-items: center;
+  transform: translateY(25px);
+}
+
+.table__content-detail {
+  display: flex;
+  flex-direction: column;
+}
+
+.table__content-pay {
+  font-weight: normal;
+}
+
+.table__content {
+  width: 35%;
+}
+
+.table__category {
+  width: 15%;
+}
+
+.table__date {
+  width: 25%;
+}
+
+.table__price {
+  width: 25%;
+}
```

이것도 좋지만 scss의 bracket을 사용할 수 있다는 점을 활용하면 더 가독성이 좋을것 같습니다 :)

```suggestion

.table{
  $.table__content-pay {   font-weight: normal; }
  $.table__content { width: 35%; }
  $.table__category { width: 15%; }
  $.table__date { width: 25%; }
  $.table__price { width: 25%; }
}

```

---

오 $ 써서 저렇게 할 수 있군요! 반영할게요!!

---

$를 빼면 작동합니다 굿굿!

---

오옹 확인했습니다 :)